fedmsg_meta_fedora_infrastructure/taskotron.py

```diff
@@ -39,5 +46,45 @@ def link(self, msg, **config):
         if msg['topic'].endswith('taskotron.result.new'):
             return msg['msg']['result'].get('log_url', '')
 
+    def objects(self, msg, **config):
+        packages = self.packages(msg, **config)
+        return set([
+            '/'.join([
+                msg['msg']['task'].get('name', ''),
+                package,
+                msg['msg']['result'].get('outcome', '')
+            ])
+            for package in packages
+        ])
+    def packages(self, msg, **config):
+        type = msg['msg']['task']['type']
+        if type == 'koji_build':
+            nvr = msg['msg']['task']['item']
+            name, version, release = nvr.rsplit('-', 2)
+            return set([name])
+        elif type == 'bodhi_update':
+            alias = msg['msg']['task']['item']
+            default_url = 'https://bodhi.fedoraproject.org'
+            bodhi_url = config.get('bodhi_url', default_url)
+            resp = requests.get(bodhi_url + '/updates/' + alias)
+            if not bool(resp):
+                log.warn("Failed to talk to bodhi %r %r" % (resp, resp.url))
+                return set()
+            data = resp.json()
+            builds = data['update']['builds']
+            nvrs = [build['nvr'] for build in builds]
+            packages = [nvr.rsplit('-', 2)[0] for nvr in nvrs]
+            return set(packages)
+
+        log.warn('Unhandled taskotron type %r' % type)
```

Comment:
If you see this, please let us know and we'll send a patch. But currently I think we only publish `koji_build` and `bodhi_update`.